<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UdpFile</class>
 <widget class="QDialog" name="UdpFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Udp-file</string>
  </property>
  <widget class="QPushButton" name="btn_openfile">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>490</y>
     <width>187</width>
     <height>57</height>
    </rect>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_update">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>490</y>
     <width>187</width>
     <height>57</height>
    </rect>
   </property>
   <property name="text">
    <string>更新程序</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>148</y>
     <width>105</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>本地IP:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_localip">
   <property name="geometry">
    <rect>
     <x>209</x>
     <y>140</y>
     <width>294</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>148</y>
     <width>135</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>本地端口:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_localport">
   <property name="geometry">
    <rect>
     <x>662</x>
     <y>140</y>
     <width>294</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>230</y>
     <width>105</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>远方IP:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_remoteip">
   <property name="geometry">
    <rect>
     <x>209</x>
     <y>220</y>
     <width>294</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>230</y>
     <width>135</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>远方端口:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_remoteport">
   <property name="geometry">
    <rect>
     <x>662</x>
     <y>220</y>
     <width>294</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="bar_sendprogress">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>630</y>
     <width>911</width>
     <height>32</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_openfile</sender>
   <signal>clicked()</signal>
   <receiver>UdpFile</receiver>
   <slot>slotOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_update</sender>
   <signal>clicked()</signal>
   <receiver>UdpFile</receiver>
   <slot>slotUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>703</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>721</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_localip</sender>
   <signal>textChanged(QString)</signal>
   <receiver>UdpFile</receiver>
   <slot>slotLocalIP(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_localport</sender>
   <signal>textChanged(QString)</signal>
   <receiver>UdpFile</receiver>
   <slot>slotLocalPort(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>821</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>846</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_remoteip</sender>
   <signal>textChanged(QString)</signal>
   <receiver>UdpFile</receiver>
   <slot>slotRemoteIP(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_remoteport</sender>
   <signal>textChanged(QString)</signal>
   <receiver>UdpFile</receiver>
   <slot>slotRemotePort(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>912</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>914</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotUpdate()</slot>
  <slot>slotOpen()</slot>
  <slot>slotLocalIP(QString)</slot>
  <slot>slotLocalPort(QString)</slot>
  <slot>slotRemoteIP(QString)</slot>
  <slot>slotRemotePort(QString)</slot>
  <slot>slotProgressBar(int)</slot>
 </slots>
</ui>
